homeassistant/components/media_player/cast.py

```diff
@@ -68,12 +68,17 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     casts = []
 
+    all_chromecasts = pychromecast.get_chromecasts()
```

Comment:
Yeh that's true, this only happens at startup, and the two calls will only happen if neither auto discovery or hard-coded host names are being used, so it's not a big deal.